docs/installation/aws-eks.md

```diff
@@ -27,11 +27,14 @@ If you use AWS SSO on your account, then you can also configure SSO for AWS CLI
   export CAPACT_NAME={name_of_the_environment}
```

Comment:
Please fix indent for this block. As you can see, the numbering is not continued.
![Screenshot 2021-08-05 at 14 37 07](https://user-images.githubusercontent.com/17568639/128350982-e2dd04fb-1aab-4cc8-be50-9b02e82a13c8.png)